compiler-rt/lib/builtins/aarch64/sme-libc-mem-routines.S

```diff
@@ -6,7 +6,7 @@
 
 #include "../assembly.h"
 
-#ifdef __aarch64__
+#if defined(__aarch64__) && __ARM_FP != 0
```

Comment:
Unless I've missed something, only memset uses the q registers? So memcpy should still be fine?

Reply:
Yeah, I've updated the patch after adapting some tests to make sure these all work on non-FPU hardware. That leaves only `__arm_sc_memset`.